src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    private static final int NUMBER_BOUND = 101;
+    private Random randomInt = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    // below method which provides Ball (randomColor, randomIndex);
+    Ball getRandomBall() {

```

Comment:
Please do not forget about access modifiers
```suggestion
    public Ball getRandomBall() {
```

Reply:
ok, done. I think I'll learn more, how and when use access modifiers.